python/marvin/__init__.py

```diff
@@ -412,6 +417,10 @@ def _check_config(self):
             # this toggles to latest DR when switching to public
             warnings.warn('Release {0} is not in the allowed releases.  '
                           'Switching to {1}'.format(self.release, latest), MarvinUserWarning)
+
+            # This is clunky but if we don't set self._release then setRelease can fail
+            # when it is called for the first time because the tree has not yet been planted.
+            self._release = latest
```

Comment:
`_checkPaths` should really never get called in practice.  `check_manga_dirs` calls it and that only gets called after the tree is setup